### PR TITLE
Fix rendering on smaller devices

### PR DESCRIPTION
**Pull request by ozan**
@aleksharma12 can you review please?

This PR improves the book layout in order to be usable on phones and tablets. On smaller devices, the table of contents now appears below the content, in columnar form:

screenshot 2015-12-22 10 47 12

screenshot 2015-12-22 10 48 24

screenshot 2015-12-22 10 48 51

screenshot 2015-12-22 10 49 13

This PR also removes the back/forward arrow navigation and replaces them with a single "next" link below the section content, per @aleksharma12 's suggestion.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,3 +6,5 @@ Practical Algorithms and Data Structures
 [Practical Algorithms and Data Structures](http://bradfield.institute/algos) is a book originally adapted from _Problem Solving with Algorithms and Data Structures Using Python_ which was written by Brad Miller and David Ranum and published under a Creative Commons license.
 
 To build, run `./run build-book`. Publishing occurs automatically via Travis.
+
+To develop, first install wach and rld globally: `npm install -g wach rld` then `./run dev`.
```
Comment (smart-alek) [inline README.md:10]: Nice change. My `R` key thanks you.

File: layouts/chapter.html [hunk 2]
```diff
@@ -14,51 +14,61 @@
 </head>
 <body>
 
-<% var prevItem, nextItem %>
```
Comment (smart-alek) [inline layouts/chapter.html:17]: NICE. Now we're not living in the past, eh? :clock1: :+1:

**Comment by smart-alek** [comment]
Can't speak to all of the CSS changes ( :grimacing: ), but I checked out the Heroku page on my iPhone6 and was met with Utter Madness. See below for the grisly details.

Looked quite nice on several size settings on my Macbook Air, though!

img_0128

**Comment by ozan** [comment]
Hmm, interesting. Glad you checked :). Works at that size in the emulator, and works on my physical Android phone, so I'm guessing it's a mobile safari quirk. Will look.

**Comment by ozan** [comment]
Ah, actually just Safari generally :)

screen shot 2016-01-05 at 4 32 10 pm

**Comment by ozan** [comment]
@aleksharma12 can you give that one a try on your iphone? Tested and confirmed fixed on desktop safari.

**Comment by smart-alek** [comment]
Looks great; now I can study algorithms on the toilet. :toilet: :+1:

Ship it when you're ready!

EDIT: Found an interesting bug while reviewing #34 and pre boxes:

If I navigate to the "Introduction to Algorithm Analysis" from the homepage on iPhone 6 Safari, it only loads a top chunk of the page. If I then hit back, it cuts off a similar size at the bottom of the page.

I **believe** that this is only happening on pages with pre boxes. Since this PR is still an improvement over what we currently have, it's up to you whether you want to ship or not.

**Comment by ozan** [comment]
@aleksharma12 thanks for the review!

> Found an interesting bug while reviewing #34 and pre boxes:

Oh I think that might actually be _fixed_ when combining these changes and #34. So I'm going to go ahead and merge this, then once #34 is also merged we can see if it's still an issue.

**Comment by ozan** [comment]
Yeah I merged this (via master) into #34 and looks like that issue should be fixed.

**Pull request merged**